pkg/services/alerting/notifiers/pushover.go

```diff
@@ -322,12 +331,16 @@ func (pn *PushoverNotifier) genPushoverBody(evalContext *alerting.EvalContext, m
 	}
 
 	// Add priority
-	err = w.WriteField("priority", strconv.Itoa(pn.Priority))
+	priority := pn.AlertingPriority
+	if evalContext.Rule.State == models.AlertStateOK {
+		priority = pn.OkPriority
```

Comment:
```suggestion
		priority = pn.OKPriority
```

Reply:
Ahh appreciate it thanks, I did notice that but was going to wait until I was back near my computer.